internal/pkg/cli/dialog/create.go

```diff
@@ -12,13 +12,11 @@ import (
 	createConfig "github.com/keboola/keboola-as-code/pkg/lib/operation/project/local/create/config"
 	createRow "github.com/keboola/keboola-as-code/pkg/lib/operation/project/local/create/row"
 	createBranch "github.com/keboola/keboola-as-code/pkg/lib/operation/project/remote/create/branch"
-	loadState "github.com/keboola/keboola-as-code/pkg/lib/operation/state/load"
 )
 
 type createDeps interface {
 	Options() *options.Options
 	StorageApi() (*storageapi.Api, error)
-	ProjectState(loadOptions loadState.Options) (*project.State, error)
 }
 
```

Comment:
V tomto commite su zmeny v `dialog` pkg.
Tj. project state sa nenacitava v `dialog` ale je to parameter.